ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/VolatileDB/Impl.hs

```diff
@@ -189,7 +189,7 @@ openDB ::
      , VolatileDbSerialiseConstraints blk
      )
   => VolatileDbArgs Identity m blk
-  -> (forall st. WithTempRegistry st m (VolatileDB m blk, st) -> ans)
+  -> (forall st. WithTempRegistry st m ((VolatileDB m blk, MaxSlotNo), st) -> ans)
```

Comment:
Why add this here, instead of calling `getMaxSlotNo` where ncessary?

Reply:
To have it traced in the log of the node

Reply:
But instead of returning this as an extra argument here, couldn't you call `getMaxSlotNo` where you trace `OpenedVolatileDB`? Eg suppose we might want to trace even more stuff in `OpenedVolatileDB` in the future, should it then also be added to `openDB`?

Reply:
Ah I see your point now

Reply:
Done